livereload.js

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
+const { spawn } = require("child_process");
+const { normalize } = require("path");
+
+const termiateSignals = ["close", "exit", ""];
+
+function startSubprocess() {
+  return new Promise((res, rej) => {
+    const proc = spawn(process.argv.slice(2).map(normalize).join(" "), {

```

Comment:
On macOS, child_process strictly wants the command and args to be separated; I got an ENOENT error when I ran the original script:

```
(node:46437) UnhandledPromiseRejectionWarning: Error: spawn node_modules/.bin/qode dist/index.js ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:268:19)
    at onErrorNT (internal/child_process.js:468:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:46437) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:46437) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Could you therefore change the spawn invocation to this, which I've found does work on macOS:

```js
    const [command, ...args] = process.argv.slice(2).map(normalize);
    const proc = spawn(
      command,
      args,
      {
        stdio: ["pipe", "pipe", "pipe"],
        shell:
          process.platform === "win32" ||
          /^(msys|cygwin)$/.test(process.env.OSTYPE),
      }
    );
```

Reply:
This should be done. I took the liberty to only normalize the command as referenced [here](https://github.com/kentcdodds/cross-env/blob/3edefc7b450fe273655664f902fd03d9712177fe/src/command.js#L29) iin cross-env.

Reply:
I think that's saying that it shouldn't be done on command argument without care. If the command arguments include any paths, each of those paths will still need to be normalised. The PR comment shows that it would break any non-path args passed in, like URLs. So we need to apply it with care – I don't have a good solution for this apart from examining the CLI args parser from Node itself (which is the same as qode) and maybe doing something with that. Otherwise, we can just normalise the first arg, which we know is a path, and leave the rest as-is (as if they are ever introduced, it's because the user has edited the template, and at that point, it's their problem to solve!).